organization/views.py

```diff
@@ -298,22 +298,30 @@ def get_success_url(self):
         return reverse('edit-organization', kwargs={'slug': self.kwargs['organization_id']})
 
 
-class OrganizationDeleteCompositionView(LoginRequiredMixin, DeleteView):
+class OrganizationDeleteCompositionView(LoginRequiredMixin, BaseDeleteRelationshipView):
     model = Composition
     template_name = 'organization/delete-composition.html'
 
-    def get_context_data(self, **kwargs):
-        context = super().get_context_data(**kwargs)
-        context['organization'] = Organization.objects.get(uuid=self.kwargs['organization_id'])
-        return context
+    def get_cancel_link(self):
+        return reverse_lazy(
+            'edit-organization-composition',
+            kwargs={
+                'organization_id': self.kwargs['organization_id'],
+                'pk': self.kwargs['pk']
+            }
+        )
+
+    def get_objects_to_delete(self):
+        composition = self.get_object()
+        parent = composition.parent.get_value().value
+        child = composition.child.get_value().value
+        return parent, child
```

Comment:
While working on this PR, I noticed that the Delete views for each entity type were all translated improperly. They were largely identical, yet each view had a separate template. Instead of fixing each template separately, I refactored them slightly to use parent Delete views, represented by `sfm_pc.base_views.BaseDeleteview` and `sfm_pc.base_views.BaseDeleteRelationshipView`.

For docs on these views, including the methods they require that are implemented in this code block, see the code below.